libraries/AC_Avoidance/AP_OAPathPlanner.h

```diff
@@ -60,7 +60,16 @@ class AP_OAPathPlanner {
         OA_OPTION_WP_RESET = (1 << 0),
     };
 
+    // enumeration for _BENDY_TYPE parameter
+    enum OABendyType {
+        OA_BENDY_HORIZONTAL = 1,
+        OA_BENDY_VERTICAL   = 2,
+    };
+
     uint16_t get_options() const { return _options;}
+    
+    uint16_t get_bendy_type() const {return _bendy_type;}
```

Comment:
Let's add a comment above get_bendy_type and if possible have it reutnr an OABendyType instead of a uint16